src/koans/15_destructuring.clj

```diff
@@ -8,37 +8,40 @@
 
 (meditations
   "Destructuring is an arbiter: it breaks up arguments"
-  (= __ ((fn [[a b]] (str b a))
+  (= ":bar:foo" ((fn [[a b]] (str b a))
          [:foo :bar]))
 
   "Whether in function definitions"
   (= (str "An Oxford comma list of apples, "
           "oranges, "
           "and pears.")
-     ((fn [[a b c]] __)
+     ((fn [[a b c]] (str "An Oxford comma list of " a ", " b ", and " c "."  ))
       ["apples" "oranges" "pears"]))
 
   "Or in let expressions"
   (= "Rich Hickey aka The Clojurer aka Go Time aka Lambda Guru"
      (let [[first-name last-name & aliases]
            (list "Rich" "Hickey" "The Clojurer" "Go Time" "Lambda Guru")]
-       __))
+       (str first-name " " last-name ((fn [[a b c]] (str " aka " a " aka " b " aka " c)) aliases))))
```

Comment:
More elegant solution would be to use [apply](https://clojuredocs.org/clojure.core/apply) and [interpose](https://clojuredocs.org/clojure.core/interpose) functions. For example:  `(str first-name " " last-name " aka " (apply str (interpose " aka " aliases)))))`

Reply:
Interpose is what I was looking for. Thank you :)